### PR TITLE
Do not directly use a version instead refer to it. (Fixes #198)

### LINKED ISSUE #198: For commit-less versions need to say how to refer to a version.
We cannot currently release new versions without creating commits because the documentation has to be updated to reflect the latest version available.

### PR DESCRIPTION
**Pull request by XerTheSquirrel**
This does not directly use a version so that we do not need to make commits to update the README every time we want to do it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,10 +42,17 @@ Your `pom.xml` file may be modified to include the following dependency:
 <dependency>
   <groupId>com.iopipe</groupId>
   <artifactId>iopipe</artifactId>
-  <version>1.9.0</version>
+  <version>(replace with latest version of IOpipe)</version>
```
Comment (pselle) [inline README.md]: I think a convention here would be something like LATEST_VERSION

File: README.md [hunk 2]
```diff
@@ -42,10 +42,17 @@ Your `pom.xml` file may be modified to include the following dependency:
...
 </dependency>
 ```
 
+To obtain the latest version you may check it at these locations, the format
```
Comment (pselle) [inline README.md]: => "Find the latest version here:"

File: README.md (hunk 1 above)
Comment (XerTheSquirrel) [reply README.md]: I know a convention that could be used to imply that it is something to be replaced by the user.

File: README.md (hunk 2 above)
Comment (XerTheSquirrel) [reply README.md]: I have updated this.

**Pull request merged**